frontend/src/container/SideNav/menuItems.tsx

```diff
@@ -16,6 +16,7 @@ import {
 	ScrollText,
 	Settings,
 	Slack,
+	// Unplug,
```

Comment:
Remove commented code.

Reply:
This is required, we need to add back the integrations once the BE is ready.